test/browser-only/constructLayoutEngine-browser.test.js

```diff
@@ -868,14 +868,24 @@ describe(`constructLayoutEngine browser`, () => {
 
     it(`works with error handlers defined in HTML`, async () => {
       const { routerElement } = parseFixture("error-handlers.html");
+
+      let numMainContentErrorMounts = 0,
+        numMainContentUnmounts = 0;
+
       const data = {
         errors: {
-          headerError: {
+          mainContentError: {
```

Comment:
I wanted the mainContentError to be a parcel because that's the one I needed to ensure that we were calling the unmount lifecycle for.